pkgs/servers/sql/postgresql/pgjwt/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchgit}:
+stdenv.mkDerivation rec {
+  name="pgjwt-${version}";
+  version="0.0.1";
+  src = fetchgit {
```

Comment:
(Using source archive that is, `fetchFromGitHub` is just a convenient way of doing it).